### ISSUE TITLE
FR: Freeze current view

### ISSUE THREAD

**Issue by SkepticMystic** [SkepticMystic/graph-analysis#35]
### Discussed in https://github.com/SkepticMystic/graph-analysis/discussions/33

**Comment by SkepticMystic** [comment]
Implemented in 0.10.0

**Comment by FelipeRearden** [comment]
> Implemented in 0.10.0

Working on version `0.10.0` in Obsidian `0.12.19`

I have one little comment to share with you:

The frozen status is lost when switching between types and coming back
- It's easier to explain following my steps:

Step 1: Open Graph Panel in `Co-Citations`
Step 2: apply frozen 
Step 3: change to `Adamic Adar` type (adamic is unfrozen by default which is 100% right!)
Step 4: get back to `Co-Citations` it is unfrozen :(

In my humble opinion would be great to store/save the frozen/unfrozen status of a type A when switching to type B and getting back to type A 

Feel absolutely free to discard this. I was just my instinct of tester/user when using the other types 

Have a great day!

**Comment by SkepticMystic** [comment]
Ahhh interesting... so the FR is to save the frozen status on a per-type basis?
I think that would be optional, so I'd make it a setting. I can also add a cmd to free/unfreeze the current view (but that's a different thing)

**Comment by FelipeRearden** [comment]
> Ahhh interesting... so the FR is to save the frozen status on a per-type basis?

Yes!!!! I'm just stating to understand all types but my feeling is that switching between types to find unseen relations :) 

The workflow is very dynamic switching between types and the backlinks pane to search for content !

> I think that would be optional, so I'd make it a setting.

100% agree @SkepticMystic !!! It's the bestiary to implement this!

> I can also add a cmd to free/unfreeze the current view (but that's a different thing)

Would be AMAZING !!!!! For keyboard lovers this will be fantastic !

If you allow me one comment about this possible future feature :) ->

-> Only one command `Graph Analysis: Frozen/Unfrozen the Active Type` . This way We can have only one hotkey applied to all types !!!

Let me know if you want me to create an FF for that.


What a great plugin @SkepticMystic !!!!

**Comment by HEmile** [comment]
> so I'd make it a setting

I don't see why this needs to be optional, sounds hard to explain and few people would want to change this i think.

**Comment by SkepticMystic** [comment]
@HEmile I guess I agree with it not needing to be optional.

And yes, it would require saving the latest results of each component
